nucypher/blockchain/eth/actors.py

```diff
@@ -1022,6 +1061,32 @@ def deposit(self, amount: int, lock_periods: int) -> Tuple[str, str]:
                                                         call_data=Web3.toBytes(lock_periods))
         return receipt
 
+    def _divide_stake(self, stake_index: int, additional_periods: int, target_value: int) -> Tuple[str, str]:
```

Comment:
I thought to have two separate functions - one is high-level that takes `stake` and other - *low-level* function only to maintain intermediary contract logic (right now it's just placeholder) so put there index instead of `stake`